src/app/public/modules/colorpicker/colorpicker-input.directive.ts

```diff
@@ -247,7 +247,7 @@ export class SkyColorpickerInputDirective
     this.skyColorpickerInput.setColorFromString(output);
 
     this.renderer.setElementStyle(element, 'background-color', setElementValue);
-    this.renderer.setElementStyle(element, 'color', setElementValue);
+    this.renderer.setElementStyle(element, 'color', 'transparent');
```

Comment:
Weird! I feel like this input should have never had a color set, because you wouldn't want to ever see that text inside the picker button. Should we just achieve this with a CSS declaration instead of doing this programmatically?

Reply:
Good call. I've updated this to use a css class and removed the setting line all together.